quantization/tests/metrics.rs

```diff
@@ -9,3 +9,11 @@ pub fn l2_similarity(v1: &[f32], v2: &[f32]) -> f32 {
         .map(|(a, b)| (a - b).powi(2))
         .sum()
 }
+
+pub fn l1_similarity(v1: &[f32], v2: &[f32]) -> f32 {
+    v1.iter()
+        .copied()
+        .zip(v2.iter().copied())
+        .map(|(a, b)| (a - b).abs())
+        .sum()
```

Comment:
We don't have to copy explicitly:

```suggestion
    v1.iter().zip(v2).map(|(a, b)| (a - b).abs()).sum()
```

We can do the same with the `l2_similarity` function.